assets/quick-buy.css

```diff
@@ -0,0 +1,197 @@
+.quick-buy-modal {
+  box-sizing: border-box;
+  opacity: 0;
+  position: fixed;
+  visibility: hidden;
+  z-index: -1;
+  margin: 0 auto;
+  top: 0;
+  left: 0;
+  overflow: auto;
+  width: 100%;
+  background: rgba(var(--color-foreground), 0.2);
+  height: 100%;
+}
+
+.quick-buy-modal[open] {
+  opacity: 1;
+  visibility: visible;
+  z-index: 101;
+}
+
+.quick-buy-modal__content {
+  border-radius: var(--popup-corner-radius);
+  background-color: rgb(var(--color-background));
```

Comment:
Wondering if the modal should follow the section's color scheme: ![](https://screenshot.click/03-14-i72bo-h5ojr.png)

Reply:
I don't think we should and AFAIK, it's not in the plan for MVP at least.

I see modals as their own component and I feel like they should stay consistent across the theme. This same modal is being used for product media, product popup block and quick buy. I think it makes sense for them to stay consistent.